UniversityofHelsinki/Styleguide
# Add Whatsapp icon

## Description
Adds Whatsapp icon to the styleguide.

## How to test
1. Run ```gulp serve```.
2. Open http://localhost:3000/#section-3-7-6. Verify you see button green button with Whatsapp icon.

## Screenshots

![1](https://user-images.githubusercontent.com/3032567/44085622-c19f8608-9fc2-11e8-8ab9-c01f77523342.jpg)
